src/routes/uni.js

```diff
@@ -2,7 +2,8 @@ const express = require('express')
 const uni = express.Router()
 const uniController = require('../controllers/uni')
 const reportController = require('../controllers/report')
-const { cacheResponse, isModOrHigher, cache } = require('../utils/helpers')
+const { cache } = require('../utils/cache')
```

Comment:
i didn't realize this file existed lol

Reply:
Yeah. Actually it's a bit confusing because the other helper should I also live there.